cypress/e2e/newsStory.cy.js

```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+
+context('News Story', () => {
+  it('News Story page renders with navigation back to parent story list', () => {
+    cy.visit('/butler-health-care/stories/its-flu-shot-time/')
+
+    cy.get('#news-stories-listing-link').click()
+    cy.location('pathname').should('equal', '/butler-health-care/stories/')
+  })
```

Comment:
Just noting here that we'll want specific tests for Lovell pages, as this link is modified in the bifurcation process.

Reply:
yes definitely. I'll stub some of those out soon